clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -2557,6 +2557,23 @@ LogicalResult cir::GetMemberOp::verify() {
 
   return mlir::success();
 }
+
+//===----------------------------------------------------------------------===//
+// ExtractMemberOp Definitions
+//===----------------------------------------------------------------------===//
+
+LogicalResult cir::ExtractMemberOp::verify() {
+  auto recordTy = mlir::cast<cir::RecordType>(getRecord().getType());
```

Comment:
Can you please add a test for the verifier?

Check `clang/test/CIR/IR/invalid-vector.cir` for reference